src/main/java/application/game/Computer.java

```diff
@@ -5,28 +5,25 @@
 import tic_tac_toe.iOS_functions$easy_computer_move;
 import tic_tac_toe.iOS_functions$minimax_move;
 
-public class ComputerVsComputer extends Game {
+public class Computer {
     private final String[] board;
     private final String marker;
     private final String computerDifficulty;
 
-    public ComputerVsComputer(String[] board, String marker, String computerDifficulty) {
-        super(board);
+    public Computer(String[] board, String marker, String computerDifficulty) {
         this.board = board;
         this.marker = marker;
         this.computerDifficulty = computerDifficulty;
     }
 
-    @Override
     public String[] getBoard() {
-        if (!isCompleted()) {
-            int computerMove = getComputerMove();
-            board[computerMove] = marker;
-        }
+        int computerMove = getMove();
+        board[computerMove] = marker;
+
         return board;
     }
 
-    public int getComputerMove() {
+    public int getMove() {
```

Comment:
Cool. I might consider testing `getMove()` through `getBoard()` if that's the only caller, and then adding that private declaration to `getMove()`. That way, you've got a narrower public interface, and you're not tempting future developers to reach into the internals of your object.